mcs/class/WindowsBase/WindowsBase-net_4_x.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
FYI you can regenerate the .csproj files by running `make update-solution-files` in the repo root.

The reason why they're checked in is to eventually enable you to build the class libs with msbuild without needing automake etc. to generate the files first.